config/src/generator.rs

```diff
@@ -49,8 +49,20 @@ pub fn validator_swarm(
             network.peer_id,
             consensus_pubkey,
             1, // @TODO: Add support for dynamic weights
-            network.network_keypairs.signing_keys.public().clone(),
-            network.network_keypairs.identity_keys.public().clone(),
+            network
+                .network_keypairs
+                .as_ref()
+                .unwrap()
+                .signing_keys
+                .public()
+                .clone(),
+            network
+                .network_keypairs
+                .as_ref()
+                .unwrap()
+                .identity_keys
+                .public()
+                .clone(),
```

Comment:
if you keep this in a networkkeypairs bundle, I recommend first extracting the network keypairs (let network_keys = network.network_keypairs.as_ref().unwrap()) prior to this piece of code so you don't have such ... a pattern.

Reply:
Thanks! I've done that. I've also used expect() instead of unwrap() to print something out if these checks fail. Note that a lot of the surrounding code is unwrapping things directly (e.g. the consensus key), but I think expect() is clearer. Let me know what you think 😄